app/views/contexts/summary/block.py

```diff
@@ -59,16 +62,16 @@ def __init__(
     def get_question(
         self,
         *,
-        block_schema,
-        answer_store,
-        list_store,
-        metadata,
-        response_metadata,
-        schema,
-        location,
-        return_to,
-        return_to_block_id,
-    ):
```

Comment:
Generally speaking it seems `serialize` methods return dicts. So such a method would return a `dict` too